Loop/Extensions/Environment+Authentication.swift

```diff
@@ -0,0 +1,40 @@
+//
+//  Environment+Authentication.swift
+//  Loop
+//
+//  Created by Rick Pasetto on 8/3/20.
+//  Copyright © 2020 LoopKit Authors. All rights reserved.
+//
+
+import LocalAuthentication
+import SwiftUI
+
+public typealias AuthenticationChallenge = (_ description: String, _ completion: @escaping (Result<Void, Error>) -> Void) -> Void
+fileprivate struct UnknownError: Swift.Error { }
+
+private struct AuthenticationChallengeKey: EnvironmentKey {
```

Comment:
Pulled from the work Rick did in LoopKit

Reply:
I'm wondering why you couldn't just use that code?  Why did you need to copy it?

Reply:
My fault - I was thinking the extension would be private, but that's only LoopKit. Thanks!